stable/xray/values.yaml

```diff
@@ -43,8 +120,8 @@ xray:
     shared:
       logging:
         consoleLog:
-          enabled: {{ .Values.xray.consoleLog }}
-      jfrogUrl: "{{ tpl (required "\n\nxray.jfrogUrl is required! This allows to connect to Artifactory.\n(You can copy the JFrog URL from Admin > Security > Settings)" .Values.xray.jfrogUrl) . }}"
+          enabled: {{ .Values.global.consoleLog | default .Values.xray.consoleLog }}
```

Comment:
Let’s keep it consistent. Create a function for this as well

Reply:
Done